mocha-client-tests/addon/tests/mocha-run.js

```diff
@@ -0,0 +1,3 @@
+mocha.checkLeaks();
+mocha.globals(['jQuery','AppView', 'ExtensionOptions', 'ExtensionView', 'WebView']);
```

Comment:
There a list of default globals in extension which from Mocha perspective looks like global leaks